R/get_dataset_additional_info.R

```diff
@@ -0,0 +1,35 @@
+#' returns a list of dataset names alongwith additional information,
+#' such as the amount of resources and the date it was last updated
+#'
+#' @param dataset_name
+#'
+#' @return
+#' @export
+#'
+get_datasets_additional_info <- function(dataset_name){
+  query <- paste0("id=", dataset_name)
+  content <- phs_GET("package_show", query)
```

Comment:
How this works was changed slightly in #25 
```suggestion
  query <- list("id" = dataset_name)
  content <- phs_GET("package_show", query)
```

You might want to update your branch with `git merge origin/master` to ensure it has all the other recent changes. I think this will be the only one that will cause an issue though.